### DISCUSSION TITLE
Using SCSS to modify the default style elements.

### DISCUSSION THREAD

**Discussion by jproque** [twbs/bootstrap#30643] [General]
I have been trying for days to figure out how to properly use Bootstrap's Sass functionality but the instructions in the documentation are not thorough enough. It does properly explain how to get npm working in my text editor (I use Atom), how to get Autoprefixer, and i think you should show what the _custom.scss file should look like once everything is properly set up. Add an example to your examples page. 

In the Theming page ( https://getbootstrap.com/docs/4.0/getting-started/theming/#sass ), you briefly mention what the file structure should look like if we are not using a package manager like npm, but all of the instructions following that are based off the assumption that we are using a package manager like npm.  

I love Bootstrap and I use it all the time, but could you please help me and try to walk me through the process of using Sass to modify the style attributes? 

PS: the "our JavaScript overview page" hyperlink in the Approach page ( https://getbootstrap.com/docs/4.0/extend/approach/ ) is linked the the Approach page and does not direct you to the Javascript overview page.

**Comment by mdo** [comment]
v5 will have an updated Customize section that's coming in #29545 that may help you. I've also started a new `bootstrap-npm-starter` repo at https://github.com/twbs/bootstrap-npm-starter/ that includes a decent setup for customizing and compiling our Sass (JS parts could use some improvement there).

npm is available by installing Node.js from https://nodejs.org/en/. Install that, relaunch your Terminal, and you should have `npm` available.

If you want to download the repo and include things manually, that is still an option, but you still must install some dependencies and setup some build tooling to be able to compile Sass and more.

Does any of that sound like something you'd like to do? Happy to explain further.
